en/docs/developer-portal/test-api.md

```diff
@@ -0,0 +1,22 @@
+# Test API
+ 
+ Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API has is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test it's functionality by providing values for the required parameters. 
+
+This guide will take you through the steps to test an API in Choreo. 
+
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click  Developer Portal link on the Choreo console header. 
+    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-small}
+2. Click **APIs**.
+3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alterntively, you can click on the API and follow step 4. 
+4. Click **Try Out** from the left panel.
+5. Select the required environment from the **Environment** dropdown. 
+    ![Test Panel](./assets/img/developer-portal/test-api/test-panel.png)
+6. Select the relevant**Subscribed Application** from the dropdown that you subscribed the API to. 
```

Comment:
```suggestion
6. From the **Subscribed Application** list, select an application you want to trigger when testing the API.

     !!! info
         This list is visible only if at least one application has subscribed to your API. For more information, see [Manage Subscriptions](manage-subscription/).
```
Suggesting this revision because this way, it is not clear what the name of the dropdown is. Also, refer [1].

Also, a user who reads this guide before reading [Manage Subscriptions](../manage-subscription/) can get confused when he/she does not see this field.

[1]  https://developers.google.com/style/ui-elements#refer-to-ui-elements

Reply:
triggering the application is incorrect isn't it?

Reply:
"Invoke" then?

Reply:
No.. you don't invoke an application. You invoke the API.

Reply:
Anyway, shall we mention something which make it clear what makes the specific application you select the "relevant" one?

Should it be "From the **Subscribed Application** list, select an application you want to use when testing the API."?

Reply:
There is also a missing space in this text.

Reply:
done